### PR TITLE
Create README.md for Matter Linux Contact Sensor Example

### PR DESCRIPTION
**Pull request by lboue**
Create README.md for [Matter Linux Contact Sensor Example](https://github.com/lboue/connectedhomeip/tree/master/examples/contact-sensor-app/linux).

### CODE REVIEW

File: examples/contact-sensor-app/linux/README.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+    install the wheel files without rebuilding:
+    `pip3 install out/debug/chip_rpc_console_wheels/*.whl`
+
+-   To use the chip-rpc console after it has been installed run:
+    `chip-console -s localhost:33000 -o /<YourFolder>/pw_log.out`
+
+-   Then you can Get and Set the light using the RPCs:
+    `rpcs.chip.rpc.Lighting.Get()`
+
+    `rpcs.chip.rpc.Lighting.Set(on=True, level=128, color=protos.chip.rpc.LightingColor(hue=5, saturation=5))`
```
Comment (agners) [inline examples/contact-sensor-app/linux/README.md:134]: Can you make this a relevant example for the contact sensor?

Comment (lboue) [reply examples/contact-sensor-app/linux/README.md:134]: Yes. I forgot to correct this one.

Comment (lboue) [reply examples/contact-sensor-app/linux/README.md:134]: @agners Where can I find the pigweed documentation for Matter with the list of available methods?

Comment (agners) [reply examples/contact-sensor-app/linux/README.md:134]: I am not very familiar with the Pigweed RCP things, but it seems this is from `examples/common/pigweed/`. Maybe there is autocomplete? (along with "tab-tab" to get  a list of available rpcs?).

Comment (lboue) [reply examples/contact-sensor-app/linux/README.md]: Fixed by [8e7f35a](https://github.com/lboue/connectedhomeip/commit/8e7f35adba459e04cd6603f09142e5a503b36aa5)

File: examples/contact-sensor-app/linux/README.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
+# Matter Linux Contact Sensor Example
+
+An example showing the use of CHIP on the Linux. The document will describe how
+to build and run Matter Linux Contact Sensor Example on Raspberry Pi. This doc
```
Comment (kiel-apple) [inline examples/contact-sensor-app/linux/README.md:4]:
```suggestion
An example showing the use of CHIP on the Linux. This document will describe how
to build and run Matter Linux Contact Sensor Example on Raspberry Pi. This document
```

File: examples/contact-sensor-app/linux/README.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+-   Then you can Get the contact sensor status using the RPCs:
+    `rpcs.chip.rpc.BooleanState.Get()`
+
+## Device Tracing
+
+Device tracing is available to analyze the device performance. To turn on
+tracing, build with RPC enabled. See [Building with RPC enabled](#building).
+
+Obtain tracing json file.
```
Comment (kiel-apple) [inline examples/contact-sensor-app/linux/README.md:139]:
```suggestion
To obtain the tracing json file, run:
```

File: examples/contact-sensor-app/linux/README.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
+# Matter Linux Contact Sensor Example
+
+An example showing the use of CHIP on the Linux. The document will describe how
+to build and run Matter Linux Contact Sensor Example on Raspberry Pi. This doc
+is tested on **Ubuntu for Raspberry Pi Server 20.04 LTS (aarch64)** and **Ubuntu
+for Raspberry Pi Desktop 20.10 (aarch64)**
+
+To cross-compile this example on x64 host and run on **NXP i.MX 8M Mini**
```
Comment (kiel-apple) [inline examples/contact-sensor-app/linux/README.md:8]:
```suggestion
To cross-compile this example on an x64 host and run it on **NXP i.MX 8M Mini**
```

**Review by kiel-apple — APPROVED** [review]

**Review by agners — APPROVED** [review]

**Pull request merged**